qe/plugins/org.jboss.tools.fuse.qe.reddeer/resources/.ssh/known_hosts

```diff
@@ -0,0 +1,2 @@
+[0.0.0.0]:8101 ssh-dss AAAAB3NzaC1kc3MAAACBAP1/U4EddRIpUt9KnC7s5Of2EbdSPO9EAMMeP4C2USZpRV1AIlH7WT2NWPq/xfW6MPbLm1Vs14E7gB00b/JmYLdrmVClpJ+f6AR7ECLCT7up1/63xhv4O1fnxqimFQ8E+4P208UewwI1VBNaFpEy9nXzrith1yrv8iIDGZ3RSAHHAAAAFQCXYFCPFSMLzLKSuYKi64QL8Fgc9QAAAIEA9+GghdabPd7LvKtcNrhXuXmUr7v6OuqC+VdMCz0HgmdRWVeOutRZT+ZxBxCBgLRJFnEj6EwoFhO3zwkyjMim4TwWeotUfI0o4KOuHiuzpnWRbqN/C/ohNWLx+2J6ASQ7zKTxvqhRkImog9/hWuWfBpKLZl6Ae1UlZAFMO/7PSSoAAACAdMQdQ5QeTcig7aT2GplLf6Zv01aNBEMpo6LPmPJB1tdSdpVzivDw3Qfx7DPBEfXwSl9AKFRKUKpase5By2QvC39h8+X0AyvMa7EYRbeoIkSmQpvmm5lUPcaIvsF6G/+0HfaWUpIg+C13aAz+N8Q9iYZBYAr7zYxw5mDdRwOgTZg=
```

Comment:
ok
is it possible to provide a file explaining it? in a resources/README.txt file perhaps? What do you think?